src/main/java/edu/byu/cs/autograder/git/GitHelper.java

```diff
@@ -142,7 +143,7 @@ private CommitVerificationResult verifyRegularCommits(Git git, Collection<Submis
 
         CommitsByDay commitHistory = CommitAnalytics.countCommitsByDay(git, lowerThreshold, upperThreshold);
         CommitVerificationResult commitVerificationResult = commitsPassRequirements(commitHistory);
-        LOGGER.debug("Commit verification result: " + JSON.toString(commitVerificationResult));
+        LOGGER.debug("Commit verification result: {}", JSON.toString(commitVerificationResult));
```

Comment:
Thanks for fixing this printing issue. What was wrong with the way I had it?

Reply:
Supposedly this change might make it faster? I think java string concatenation runs in linear time rather than constant time and this may or may not make it marginally faster. (Mostly I wanted to remove an IntelliJ warning)